openlibrary/plugins/openlibrary/js/readmore.js

```diff
@@ -29,15 +29,23 @@ export function initClampers(clampers) {
                 Clamper shows used to show more/less by toggling `hidden`
                 style on parent .clamp tag
             */
-            $(clamper).on('click', function() {
+            $(clamper).on('click', function (event) {
                 const up = $(this);
-                if (up.hasClass('clamp')) {
-                    up.css({display: up.css('display') === '-webkit-box' ? 'unset' : '-webkit-box'});
 
-                    if (up.attr('data-before') === '\u25BE ') {
-                        up.attr('data-before', '\u25B8 ')
-                    } else {
-                        up.attr('data-before', '\u25BE ')
+                // prevent the subjects from collapsing/expanding when the <a> link is being clicked
+                if (event.target.nodeName === 'A') {
+                    return
+                }
+                else {
+
+                    if (up.hasClass('clamp')) {
+                        up.css({ display: up.css('display') === '-webkit-box' ? 'unset' : '-webkit-box' });
+
+                        if (up.attr('data-before') === '\u25BE ') {
+                            up.attr('data-before', '\u25B8 ')
+                        } else {
+                            up.attr('data-before', '\u25BE ')
+                        }
                     }
                 }
```

Comment:
```suggestion
                }

                if (up.hasClass('clamp')) {
                    up.css({ display: up.css('display') === '-webkit-box' ? 'unset' : '-webkit-box' });

                    if (up.attr('data-before') === '\u25BE ') {
                        up.attr('data-before', '\u25B8 ')
                    } else {
                        up.attr('data-before', '\u25BE ')
                    }
                }
```
No need for an `else` block here, since we are returning in the `if` block on line 37.